lua/rocks/config/internal.lua

```diff
@@ -97,6 +97,29 @@ if #config.debug_info.unrecognized_configs > 0 then
     )
 end
 
+---@type
+---|"sysv"
+---| "hpux"
+---| "netbsd"
+---| "linux"
+---| "hurd"
+---| "solaris"
+---| "aix"
+---| "irix"
+---| "freebsd"
+---| "openbsd"
+---| "dragonfly"
+---| "haiku"
+---| "windows"
+---| "cygwin"
+---| "macosx"
+local current_os = assert((require("sysdetect").detect()), "[rocks]: unable to detect current operating system!")
```

Comment:
We could also check if  `vim.uv.os_uname().sysname`  is `"Windows"`.

Reply:
I did consider that but I thought that being open to more accurate system detection wouldn't hurt (just in case some random OS decides to have a different PATH separator lol). This rock can also detect architecture which we may (?) need.

Reply:
I would say YAGNI for now. But it's good to know about the package in case we end up needing it 😀